EIPS/eip-5851.md

```diff
@@ -0,0 +1,256 @@
+
+- Claims: Decentralised Identity(DID) Claims are a set of assertions that the DID Holder must fulfill to be verified.
+
+- Holder: The holder is the entity that stores the claim, such as a digital identity provider or a DID registry. The holder is responsible for validating the claim and providing verifiable evidence of the claim.
+
+- Claimer: A claimer is the party making a claim, such as in an identity verification process.
+
+- Issuer: An entity creates a verifiable credential from claims about one or more subjects to a holder. Example issuers include governments, corporations, non-profit organizations, trade associations, and individuals.
+
+- Verifier: A verifier in DID is an entity that validates data provided by an issuer of verifiable credentials, determining its accuracy, origin, currency and trustworthiness.
+
+
+
+### Metadata Standard
+
+Claims MUST be stored in the following structures:
+
+#### 1. Metadata information
+
+Each claim requirement MUST be exposed using the following struct:
+
+```solidity
+    /** Metadata
+    * 
+    * @param title defines the name of the claim field
+    * @param kind is the nature of the data (bool,string,address,bytes,..)
+    * @param description additional information about claim details.
+     */
+    struct Metadata {
+        string title;
+        string kind;
+        string description;
+    }
+```
+
+### 2. Values Information
+
+This structure will be used to define the actual claim information, based on the description of the `Metadata` structure, the structure is similar to `Values` structure of [EIP-3475](./eip-3475.md).
+
+### 3. Claim structure
+
+Claims (eg. `age >= 18`, jurisdiction in allowlist, etc.) are represented by one or many instances of the `Claim` structure below:
+
+```solidity
+    /** Claims
+    * 
+    * Claims structure consist of the conditions and value that holder claims to associate and verifier has to validate them.
+    * @notice the below given parameters are for reference purposes only, developers can optimize the fields that are needed to be represented on-chain by using schemes like TLV, encoding into base64 etc.
+    * @dev structure that defines the parameters for specific claims of the SBT certificate
+    * @notice this structure is used for the verification process, it contains the metadata, logic and expectation
+    * @notice logic can represent either the enum format for defining the different operations, or they can be logic operators (stored in form of ASCII figure based on unicode standard). like  e.g: 
+("⊄" = U+2284, "⊂" = U+2282,  "<" = U+003C , "<=" = U + 2265,"==" = U + 003D, "!="U + 2260, ">=" = U + 2265,">" =  U + 2262).
```

Comment:
I still don't understand what `logic` is supposed to look like or how it is supposed to be used. Is it supposed to be parsed by other contracts? By off-chain software? Is it just a human-readable string?

Reply:
it can be used for both front-end and smart-contract logic:
in this example, the more than 18 yrs can be used by off-chain applications, such as an adult website to confirm that the wallet holder is more than 18 yrs old. secondly, when a smart contract try to read the logic, it can decipher the logic string and convert this into code logic. Again, in this example, off-chain or on-chain applications can convert this operator into code logic
`if (logic==">="){};`
`if (logic=="<"){};`

When it is on the frontend use case for human readable info,it can be converted to string that UI will be displaying for human.